packages/dds/tree/src/test/shared-tree/fuzz/composeVsIndividual.fuzz.spec.ts

```diff
@@ -148,24 +170,42 @@ describe("Fuzz - composed vs individual changes", () => {
 			DDSFuzzTestState<SharedTreeTestFactory>
 		> = {
 			workloadName: "SharedTree",
-			factory: new SharedTreeTestFactory(() => {}),
+			factory: new SharedTreeTestFactory(createOnCreate(undefined)),
 			generatorFactory,
 			reducer: fuzzComposedVsIndividualReducer,
 			validateConsistency: () => {},
 		};
 		const emitter = new TypedEventEmitter<DDSFuzzHarnessEvents>();
 		emitter.on("testStart", (initialState: BranchedTreeFuzzTestState) => {
 			initialState.main = viewFromState(initialState, initialState.clients[0]);
-			initialState.branch = initialState.main.fork() as FuzzTransactionView;
-			initialState.branch.currentSchema = initialState.main.currentSchema;
+
+			const branchCheckout = initialState.main.checkout.fork();
```

Comment:
why don't we dispose the branch, like in the similar code above?

Reply:
For these tests we keep track of two views (one for main, and one for the fork), where all of the edits are applied to the forked view. We disposed of the forked view when applying schemaOps, since the new forked view will have the updated schema.

But now that logic has been deleted, since we removed schemaOps from these tests.